Source/Core/Plane.js

```diff
@@ -166,6 +169,34 @@ define([
         return Cartesian3.dot(plane.normal, point) + plane.distance;
     };
 
+    var scratchPosition = new Cartesian3();
+    /**
+     * Transforms the plane by the given transformation matrix.
+     *
+     * @param {Plane} plane The plane.
+     * @param {Matrix4} transform The transformation matrix.
+     * @param {Plane} [result] The object into which to store the result.
+     * @returns {Plane} The plane transformed by the given transformation matrix.
+     */
+    Plane.transformPlane = function(plane, transform, result) {
```

Comment:
Make sure to write tests for this once you start writing tests.

Reply:
Also since this is a new thing it should be added to `CHANGES.md`, unless this is better off private.

Reply:
Just name this `transform` since `Plane` is the class name?